content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,90 @@
+# check-ignore
+
+Check whether any given files or directories are excluded from DVC due to the
+patterns found in [`.dvcignore`](/doc/user-guide/dvcignore).
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        File or directory paths to check (wildcards supported)
+```
+
+## Description
+
+This helper command checks whether the given `targets` are ignored by DVC
+according to the [`.dvcignore` file](/doc/user-guide/dvcignore) (if any). The
+ones that are ignored indeed are printed back.
+
+> Note that your shell may support path wildcards such as `dir/file*` and these
```

Comment:
same - it's fine to keep it here, but that what I would expect anyways since multiple targets are supported. A bit not consistent since we don't make notes like this in other commands.

Reply:
Oh I see you noticed the note. Yeah any other commands where this note would be especially relevant? add/commit/remove? status/repro? fetch/pull/push? metrics/plots? (un)freeze?

Reply:
I guess best to just remove it...

Reply:
Also removed in #1673